edb/lib/cal.edgeql

```diff
@@ -1094,6 +1300,12 @@ CREATE CAST FROM cal::relative_duration TO std::json {
 };
 
 
+CREATE CAST FROM cal::date_duration TO std::json {
+    SET volatility := 'Stable';
```

Comment:
Can't it be immutable? ditto for other json<>date_duration; they're independent of current db config

Reply:
Postgres' `stable` marking is due to configurability of timezone and interval output format.  EdgeDB has stable timezone (`UTC`), and stable output format, so we should be able to mark these as immutable.